suite-common/test-utils/src/conditionalDescribe.ts

```diff
@@ -1,11 +1,7 @@
-export const conditionalDescribe = (
-    skipCondition: boolean,
-    title: string,
-    fn: jest.EmptyFunction,
-) => {
-    if (skipCondition) {
-        describe.skip(title, fn);
-    } else {
+export const conditionalDescribe = (condition: boolean, title: string, fn: jest.EmptyFunction) => {
+    if (condition) {
```

Comment:
Would it make sense to change `condition` to `platforms` and evaluate here? It seems we currently don't have anything else than platform conditions.

Reply:
Good idea, but the function is generic on purpose. It was discussed with desktop team, that they might use it as well later when needed. For that reason it is placed to `suite-common` rather than `suite-native`.